### PR TITLE
adding article about airdropping memberships

### PR DESCRIPTION
**Pull request by julien51**
# Description

All in the title!

# Issues

Fixes #
Refs #

# Checklist:

- [ ] 1 PR, 1 purpose: my Pull Request applies to a single purpose
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have updated the docs to reflect my changes if applicable
- [ ] I have added tests (and stories for frontend components) that prove my fix is effective or that my feature works
- [ ] I have performed a self-review of my own code
- [ ] If my code involves visual changes, I am adding applicable screenshots to this thread

### CODE REVIEW

**Review by ccarfi — APPROVED** [review]

File: unlock-protocol-com/blog/airdrop-nft-memberships.md [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+
+Once your lock is deployed, you can head out to the members page.
+
+![Dashboard](/images/blog/airdrop-nft-memberships/dashboard.png)
+
+On that page you will see the list of all the members on your lock. You can also see a button that lets you airdrop an NFT: click on it!
+
+![Members Page](/images/blog/airdrop-nft-memberships/members-page.png)
+
+A drawer will open and let you easily airdrop a membership to any address.
```
Comment (desertrat10) [inline unlock-protocol-com/blog/airdrop-nft-memberships.md:32]: [what's a drawer- a window? ]  Once the "drawer" opens, you can airdrop a membership to any address you paste into that space/drawer.

File: unlock-protocol-com/blog/airdrop-nft-memberships.md [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+
+- Make sure the lock is the correct one!
+
+- You need to enter the recipient's address. The field supports ENS names, in order to avoid making typos!
```
Comment (desertrat10) [inline unlock-protocol-com/blog/airdrop-nft-memberships.md:36]: Enter the recipient's address or ENS name. Using the ENS name will help avoid many common typos.

File: unlock-protocol-com/blog/airdrop-nft-memberships.md [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+
+- Next you can select when this airdropped membership is going to expire. By default, it is based on the duration of the lock, but can be customized. This flexibility means you can grant a key for a few minutes for trials!
```
Comment (desertrat10) [inline unlock-protocol-com/blog/airdrop-nft-memberships.md:38]: After you've entered the address, select the expiration date on the airdropped membership. By default, the expiration is the same as the duration of the lock, but you can customize the airdropped NFT expiration date. This feature works great for granting temporary access keys for trials and demos.

File: unlock-protocol-com/blog/airdrop-nft-memberships.md [hunk 4]
```diff
@@ -0,0 +1,58 @@
...
+
+- Finally you can select a "key manager". At Unlock, we decouple the "owner" of an NFT from the account who has the ability to transfer said NFT. Most of the time, and by default, it is the same... but in some cases, it can be important to make sure that someone, even if they own an NFT is not able to transfer it. That's the case for credit card purchases. For security reasons, we want to make sure that NFT "bought" this way are not transferable. Similarly, granted keys can be "cancelled" by their owner, triggering a refund, by default. So it can be important to retain the key manager role.
```
Comment (desertrat10) [inline unlock-protocol-com/blog/airdrop-nft-memberships.md:40]: Last, but never least, you may delegate a "key manager". At Unlock, we decouple the "owner" of an NFT from the account who possess transfer abilities for the NFT in question. By default and in many cases, the owner and manager are the same. But certain unique use-cases do require the blocking of transfer access by the NFT owner. For security reasons, we want to ensure that the NFT "bought" this way are non-transferable. Similarly, granted keys can be "cancelled" by their owner, and by default, triggering a refund. In some cases, retaining the key manager role is important.

File: unlock-protocol-com/blog/airdrop-nft-memberships.md [hunk 5]
```diff
@@ -0,0 +1,58 @@
...
+
+- Once ready, submit the transaction!
+
+![Airdrop](/images/blog/airdrop-nft-memberships/dashboard.gif)
+
+## Using code
+
+Of course, granting keys is possible through code! The `grantKeys` function on the lock can be called by any _lock manager_ or _key granter_ ([read more about the roles](https://docs.unlock-protocol.com/developers/smart-contracts/lock-api/access-control)). In order to reduce risks, we recommend using a dedicated `key granter` in your application so that if that account (or contract) was compromised, the attacker could "only" grant new keys (which could be cancelled by a lock manager too...).
```
Comment (desertrat10) [inline unlock-protocol-com/blog/airdrop-nft-memberships.md:48]: Don't forget, you can always grant keys through code! 

To reduce risks, we recommend using a dedicated `key granter` in your application. That way, if an account (or contract) is compromised, the attacker can "only" grant new keys (which the lock manager can cancel).

File: unlock-protocol-com/blog/airdrop-nft-memberships.md [hunk 6]
```diff
@@ -0,0 +1,58 @@
...
+
+Then, on your code you would call the [`grantKeys` function](https://docs.unlock-protocol.com/developers/smart-contracts/lock-api#grantkeys). This function can in fact be called to grant multiple keys to multiple users at once. There is a practical limit based on the block-size of each chain. You would call this function with 3 arrays of the same size:
```
Comment (desertrat10) [inline unlock-protocol-com/blog/airdrop-nft-memberships.md:50]: You can call this function to grant multiple keys to multiple users at the same time.

File: unlock-protocol-com/blog/airdrop-nft-memberships.md [hunk 7]
```diff
@@ -0,0 +1,58 @@
...
+
+1. the recipients
+2. the expiration timestamps
+3. the key managers
+
+---
+
+Airdropping memberships is a very important part of the Unlock protocol because it enables not only multiples types of memberships, but also new ways of rewarding or incentivizing community members!
```
Comment (desertrat10) [inline unlock-protocol-com/blog/airdrop-nft-memberships.md:58]: Airdropping is a vital function of the protocol as it enables both multiple types of memberships and new ways of rewarding or incentivizing community members!

File: unlock-protocol-com/blog/airdrop-nft-memberships.md (hunk 1 above)
Comment (julien51) [reply unlock-protocol-com/blog/airdrop-nft-memberships.md:32]: See video :)

**Comment by julien51** [comment]
Thanks!

**Pull request merged**